spec/models/concerns/digital_object_concerns/export_fields_behavior_spec.rb

```diff
@@ -58,5 +58,28 @@
         expect(exported_xml.css("mods|titleInfo[lang]", xmlns).map { |t| t['lang'] }).to eql([title.dig('value_lang', 'tag')])
       }
     end
+    context "with language-tagged field data" do
+      include_context 'with english-adjacent language subtags'
+      include_context 'with stubbed search adapters'
+      let(:lang) { 'sco' }
+      let(:script) { 'Latn' }
+      let(:lang_tag) { "#{lang}-#{script}" }
+      let(:value_lang) { { 'tag' => lang_tag } }
+      let(:value) { 'Ane Pleasant Satyre of the Thrie Estaitis' }
+      let(:field_export_profile) { FactoryBot.create(:field_export_profile, :for_alternative_title_field) }
+      let(:xmlns) { { 'mods' => 'http://example.org/' } }
+      let(:item) { FactoryBot.create(:item, :with_ascii_dynamic_field_data, value: value, value_lang: value_lang) }
+      # reload item for rehydrated term/language data
+      let(:loaded_item) { DigitalObject::Item.find(item.id) }
+      let(:exported_data) { loaded_item.render_field_export(field_export_profile) }
+      let(:exported_xml) { Nokogiri::XML(exported_data) }
+      before { FactoryBot.create(:export_rule, :for_alternative_title_field, field_export_profile: field_export_profile, dynamic_field_group: DynamicFieldGroup.first) }
+      it {
```

Comment:
```suggestion
      it 'produces the expected XML' {
```
Related: I expected rubocop to complain here about using a curly brace for a multi-line block.

Reply:
The above suggestion is optional, btw.  Just personal preference.